#! /bin/sh -e
## 01_defaults.debian.dpatch by Tollef Fog Heen <tfheen@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Set sane defaults in Defaults.py.in
## DP: 

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch ${2:+-d $2}}"

if [ $# -lt 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch $patch_opts -p1 < $0;;
    -unpatch) patch $patch_opts -p1 -R < $0;;
    *)
        echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
        exit 1;;
esac

exit 0
@DPATCH@

--- mailman-2.1.4.orig/Mailman/Defaults.py.in
+++ mailman-2.1.4/Mailman/Defaults.py.in
@@ -54,7 +54,7 @@
 # disable Mailman's logo footer altogther, hack
 # Mailman/htmlformat.py:MailmanLogo(), which also contains the hardcoded links
 # and image names.
-IMAGE_LOGOS = '/icons/'
+IMAGE_LOGOS = '/doc/mailman/images/'
 
 # The name of the Mailman favicon
 SHORTCUT_ICON = 'mm-icon.png'
@@ -85,7 +85,7 @@
 # because otherwise the default mappings won't be correct.
 DEFAULT_EMAIL_HOST = '@MAILHOST@'
 DEFAULT_URL_HOST = '@URLHOST@'
-DEFAULT_URL_PATTERN = 'http://%s/mailman/'
+DEFAULT_URL_PATTERN = 'http://%s/cgi-bin/mailman/'
 
 # DEFAULT_HOST_NAME has been replaced with DEFAULT_EMAIL_HOST, however some
 # sites may have the former in their mm_cfg.py files.  If so, we'll believe
@@ -1247,7 +1247,7 @@
 MAILDIR_DIR     = os.path.join(QUEUE_DIR, 'maildir')
 
 # Other useful files
-PIDFILE = os.path.join(DATA_DIR, 'master-qrunner.pid')
+PIDFILE = "/var/run/mailman/mailman.pid"
 SITE_PW_FILE = os.path.join(DATA_DIR, 'adm.pw')
 LISTCREATOR_PW_FILE = os.path.join(DATA_DIR, 'creator.pw')
 
